array.go

```diff
@@ -360,6 +360,97 @@ func (a *Array) NonEmptyDomain() ([]NonEmptyDomain, bool, error) {
 	}
 }
 
+// NonEmptyDomainVarFromName retrieves the non-empty domain from an array for a
+// given dimension name.
+// This returns the bounding coordinates for the dimension
+func (a *Array) NonEmptyDomainVarFromName(dimName string) ([]NonEmptyDomain, []NonEmptyDomain, bool, error) {
```

Comment:
We should not need to return two lists of nonEmptyDomains here. This should just return a single `NonEmptyDomain` object no?

Reply:
Correct it is improved in https://github.com/TileDB-Inc/TileDB-Go/pull/108 just merged in this branch